docs/pages/usage/parameters.md

```diff
@@ -0,0 +1,576 @@
+
+### Program-Specific Parameters
+
+Depending on the type of program being proved (e.g., Cairo programs, custom AIRs), there may be additional parameters specific to that program type. These could include:
+
+- Input sizes
+- Memory segment configurations
+- Custom constraint parameters
+
+These characteristics would be established in accordance with the particular program needs and the AIR (Algebraic Intermediate Representation) that is being utilised.
+
+It is essential to comprehend and set up these elements appropriately in order to produce reliable and effective STARK proofs using the Stone Prover.
+
+
+## Creating a Parameter File
+
+In order for the Stone Prover to provide accurate and effective proofs, it is imperative that you create a suitable parameter file. To generate a parameter file for your particular use case, follow these steps:
+
+
+1. Determine Your Program Type
+   - Identify whether you're using a Cairo program, a custom AIR, or another type of computation.
+   - This will influence which parameters you need to include.
+
+2. Set the Field Prime
+   - Choose a prime field large enough to ensure security for your application.
+   - For most applications, you can use the default Stark252 prime:
+     ```json
+     "field_prime": 3618502788666131213697322783095070105623107215331596699973092056135872020481
+     ```
+
+3. Configure STARK Parameters
+   - Set up the `stark` object with appropriate FRI parameters:
+     ```json
+     "stark": {
+       "fri": {
+         "fri_step_list": [0, 3, 3, 3, 3, 3, 2, 1],
+         "last_layer_degree_bound": 64,
+         "n_queries": 18,
+         "proof_of_work_bits": 24
+       },
+       "log_n_cosets": 4
+     }
+     ```
+   - Adjust `fri_step_list` and `last_layer_degree_bound` based on your program's size and security requirements.
+
+4. Add Program-Specific Parameters
+   - For Cairo programs:
+     - Specify memory segment sizes, number of steps, etc.
+   - For custom AIRs:
+     - Include parameters related to your specific constraint system.
+
+5. Validate the Parameter File
+   - Ensure that the critical equation is satisfied:
+     ```
+     log₂(last_layer_degree_bound) + ∑fri_step_list = log₂(#steps) + 4
+     ```
+   - Check that all required parameters for your program type are included.
+
+6. Save the File
+   - Save your parameter file with a `.json` extension.
+   - Use a descriptive name, e.g., `my_program_params.json`.
+
+7. Test the Parameter File
+   - Run the Stone Prover with your parameter file on a small test case.
+   - Verify that the prover executes successfully and produces a valid proof.
+
+Keep in mind that your particular use case may dictate different requirements for the precise characteristics. When creating a parameter file, always check the requirements of your program and the Stone Prover manual.
+
+
+## Usage Configuration
+
+
+
+```json
+{
+  "field_prime": 3618502788666131213697322783095070105623107215331596699973092056135872020481,
```

Comment:
Try to run the prover with this parameters

Reply:
Alright no problem